sdk/core/azure-core/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 ### Features Added
 
+- Added `Azure::Core::Uuid::AsArray` and `Azure::Core::Uuid::FromArray` to enable reading or writing from an existing UUID.
```

Comment:
In other instances in the changelog, we put functions with braces, i.e. as `function()`:
```suggestion
- Added `Azure::Core::Uuid::AsArray()` and `Azure::Core::Uuid::FromArray()` to enable reading or writing from an existing UUID.
```